what-the-stack/src/main/res/layout/activity_what_the_stack.xml

```diff
@@ -95,7 +95,21 @@
             android:text="@string/share_stacktrace"
             android:textColor="?attr/colorOnSurface"
             app:icon="@drawable/ic_outline_share_24"
-            app:iconTint="?attr/colorSecondary"/>
+            app:iconTint="?attr/colorSecondary" />
+
+        <com.google.android.material.button.MaterialButton
+            android:id="@+id/launchApplication"
+            style="@style/Widget.MaterialComponents.Button.OutlinedButton.Icon"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="16dp"
+            android:layout_marginTop="2dp"
```

Comment:
While the top margin on this button is correct, the bottom margin on the button above it is too high. Please change it to 2dp for consistent spacing in between all buttons.

Reply:
actually the above Share StackTrace button has marginBottom 8dp.
Should I change margin Bottom of Share StackTrace button to 2dp too?

Reply:
Yes, that's exactly what I meant.

Reply:
done